### PR TITLE
PXC 4132 5.7.41 merge part2

### PR DESCRIPTION
**Pull request by VarunNagaraju**

### CODE REVIEW

**Comment by VarunNagaraju** [comment]
Jenkins results:
https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/381/#showFailuresLink : No new failures.
(https://github.com/Percona-Lab/jenkins-pipelines/pull/2013 fixes the OEL9 innodb_stress failures.)

Reruns of aborted platforms:
https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-pipeline/5357/
https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-pipeline/5356/

File: build-ps/debian/percona-xtradb-cluster-server-5.7.preinst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 get_pcount () {
-	PSCOUNT=$(ps -ef | grep "/usr/sbin/mysqld" | wc -l)
+	PSCOUNT=$(ps -C mysqld --no-headers | wc -l)
```
Comment (kamil-holubicki) [inline build-ps/debian/percona-xtradb-cluster-server-5.7.preinst]: I'm not sure if we should touch build-ps content. As per the process described in Conflu, let's leave this part for packaging team.

**Review by kamil-holubicki — COMMENTED** [review]
As per commit message:
```
* Bumps the XB_REQUIRED_VERSION.
* Fixes trivial conflicts in build-ps/<PLATFORM>/control files.
```
If there are any conflicts, they should be resolved in the next step. 1st step should be merge with conflicts.
But, this does not apply to build-ps, as it should be kept as-is (pxc version)

XB_REQUIRED_VERSION should be bumped as conflict resolution commit (if there is no conflict, it can be done directly in PS merge commit)

In other words, I understand that it should be 1 commit where:
1. build-ps conflicts are ignored as we use PXC version
2. XB_REQUIRED_VERSION bumped, as there are no other conflicts

**Comment by VarunNagaraju** [comment]
ASAN: https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-pipeline/5374/

**Review by venkatesh-prasad-v — APPROVED** [review]
Please merge part-1 PR before merging this PR.

**Pull request merged**